### PR TITLE
Portfolio: Finish mobile version

### PR DESCRIPTION
**Pull request by rabayarabu**
Portfolio: Finish mobile version

1. Include images
2. Edit index.html file
3. Edit style.css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,36 +26,191 @@ <h4>Rabaya Rabu</h4>
...
+                                alt="Twitter"></a></li>
+                    <li><a href=""><img src="Images/Medium.png"
+                                alt="Medium"></a></li>
+                </ul>
+            </div>
+        </section>
+        <!-- works section -->
+        <section class="works">
+            <article class="grid-container">
+                <h1>My Recent Works</h1>
```
Comment (Shisui6) [inline index.html]: Kindly update this element to use another `<hx>` element apart from `<h1>`🙏. This is for semantic reasons, as a good webpage is typically supposed to have one `<h1>` tag which represents the main title

File: index.html [hunk 2]
```diff
@@ -26,36 +26,191 @@ <h4>Rabaya Rabu</h4>
...
+                        the 1500s,
+                        when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul>
+                        <li class="css">CSS</li>
+                        <li class="html">HTML</li>
+                        <li class="bootstrap">Bootstrap</li>
+                        <li class="ruby">Ruby</li>
+                    </ul>
+                    <button type="button">See project</button>
```
Comment (Shisui6) [inline index.html]: Kindly update **all** buttons on your page to have button interaction styling when hovered and pressed. You can use the `:hover` and `:active` pseudo selectors to achieve this. When doing so, please stick to the figma design guidelines for button styling👍:
image

File: index.html [hunk 3]
```diff
@@ -26,36 +26,191 @@ <h4>Rabaya Rabu</h4>
...
+                    <li class="list1">HTML</li>
+                    <li class="list2">Bootstrap</li>
+                    <li class="list3">Ruby</li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+        </section>
+        <!-- about section -->
+        <section class="about">
+            <h1>About me</h1>
```
Comment (Shisui6) [inline index.html]: Kindly update this element to use another `<hx>` element apart from `<h1>`🙏. This is for semantic reasons, as a good webpage is typically supposed to have one `<h1>` tag which represents the main title

File: index.html [hunk 4]
```diff
@@ -26,36 +26,191 @@ <h4>Rabaya Rabu</h4>
         <!-- headline section -->
```
Comment (Shisui6) [inline index.html:26]: Kindly remove the below line in your readme file:
image
 This note is a part of the guideline template and should not be part of your final readme

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @rabayarabu, 

# Requires Changes 🍀
You have done an excellent job!👏👏
Your design is spot on 🖼️, and your code follows best practices. There are just a few more things to work on🪲

## Highlights
- Excellent design ✅
- Great use of grid and flexbox ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -26,36 +26,191 @@ <h4>Rabaya Rabu</h4>
...
+                        the 1500s,
+                        when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul>
+                        <li class="css">CSS</li>
+                        <li class="html">HTML</li>
+                        <li class="bootstrap">Bootstrap</li>
+                        <li class="ruby">Ruby</li>
+                    </ul>
+                    <button type="button">See project</button>
```
Comment (victorokoroji) [inline index.html]: - [ ] As suggested by the previous reviewer, kindly update all buttons on your page to have button interaction styling when hovered and pressed. You can use the` :hover `and `:active` pseudo selectors to achieve this. When doing so, please stick to the figma design guidelines for button styling👍:

image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @rabayarabu, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @rabayarabu, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text

 ## Required Changes ♻️
- [ ] As suggested by the previous reviewer, kindly update all buttons on your page to have button interaction styling when hovered and pressed. You can use the `:hover `and `:active `pseudo selectors to achieve this. When doing so, please stick to the Figma design guidelines for button styling👍:

Screenshot 2023-03-25 at 05 13 51

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @rabayarabu 
Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- great design
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**